src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React, { useCallback } from "react";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+
+import { currentVenueSelectorData } from "utils/selectors";
+import { venueInsideUrl } from "utils/url";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
 
 import NavBar from "components/molecules/NavBar";
+import NavSearchBar from "components/molecules/NavSearchBar";
```

Comment:
Would be good to remove the default exports from these in favour of named exports, given we're doing so much related refactoring in this PR already.